### PR TITLE
Update CME urls to be env variables

### PR DESCRIPTION
**Pull request by jwade1327**
Screen Shot 2023-09-18 at 9 16 52 PM
Screen Shot 2023-09-18 at 9 39 37 PM

### CODE REVIEW

**Comment by jwade1327** [comment]
[Dr B - Links to ICLE site on staging go to prod](https://trello.com/c/VbNp3QmC/392-dr-b-links-to-icle-site-on-staging-go-to-prod)

File: sites/drbicuspid.com/config/navigation.js [hunk 1]
```diff
@@ -83,7 +85,7 @@ module.exports = {
         modifiers: ['user'],
       },
       {
-        href: '/logout',
+        href: '/user/logout',
```
[the remark this answers is not in the document]
Comment (jwade1327) [reply sites/drbicuspid.com/config/navigation.js:88]: Throws an error if /user isn't included

File: docker-compose.yml [hunk 2]
```diff
@@ -29,6 +29,8 @@ x-env-defaults: &env
   DEBUG: ${DEBUG-}
   GAM_TRACK_API_KEY: ${GAM_TRACK_API_KEY-}
   ENABLE_NATIVE_X: ${ENABLE_NATIVE_X-false}
+  WPICLE_API_HOSTNAME: ${WPICLE_API_HOSTNAME-https://my.drbicuspid.com}
```
[the remark this answers is not in the document]
Comment (Shinsina) [reply docker-compose.yml:32]: If this is going to be shared it shouldn't have a hardset default to one site, the preferred thing to do here would be to handle this like the BRAZE_API_KEY's per site and allow for them to be set independently as I believe was the case when this was an environment variable previously (see: https://github.com/parameter1/science-medicine-group-websites/pull/361/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3L134-L139)

**Review by Shinsina — DISMISSED** [review]

File: docker-compose.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (jwade1327) [reply docker-compose.yml:32]: @Shinsina This is all going to be removed within the next few weeks, if not sooner. They only need this in place on the staging site for their testing.

**Review by B77Mills — APPROVED** [review]

**Pull request merged**